example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -848,6 +875,17 @@ describe('The S3 Ingest Granules workflow', () => {
           );
           expect(reingestExecution.asyncOperationId).toEqual(asyncOperationId);
         });
+
+        // TODO remove the glacier files via ORCA API when the API is available
```

Comment:
Same here, if there's a ticket, can we link it so we can check when this happens?